src/components/Posts/ExpandedPostItem.tsx

```diff
@@ -2,23 +2,24 @@ import {
   Box,
   Button,
   Card,
+  CardActions,
   CardContent,
-  Stack,
-  Typography,
+  CardHeader,
+  CardMedia,
   Divider,
+  IconButton,
   Link,
+  Stack,
+  Typography,
 } from "@mui/material";
-import KeyboardBackspaceIcon from "@mui/icons-material/KeyboardBackspace";
-import CardHeader from "@mui/material/CardHeader/CardHeader";
-import MoreVertIcon from "@mui/icons-material/MoreVert";
-import IconButton from "@mui/material/IconButton/IconButton";
-import CardMedia from "@mui/material/CardMedia/CardMedia";
-import CardActions from "@mui/material/CardActions/CardActions";
-import FavoriteBorderOutlinedIcon from "@mui/icons-material/FavoriteBorderOutlined";
-import FavoriteOutlinedIcon from "@mui/icons-material/FavoriteOutlined";
-import AddCommentOutlinedIcon from "@mui/icons-material/AddCommentOutlined";
-import RepeatOutlinedIcon from "@mui/icons-material/RepeatOutlined";
-import ShareOutlinedIcon from "@mui/icons-material/ShareOutlined";
```

Comment:
Okay, will look into this myself and see if it needs to be addressed

Reply:
As discussed offline, we can probably leave it the way you have it. Looking at MUI's docs, Vite probably has tree shaking built in, and either way, I don't think it's a big deal because our bundle size is probably fine anyways. This would be one of those optimizations we make when we actually need it